scripts/Auth_scripts/main.ps1

```diff
@@ -0,0 +1,39 @@
+# Requires PSVersion 5.1.18362.1171
+. ".\auth.ps1"
+. ".\getpet.ps1"
+. ".\menu.ps1"
+
+Set-StrictMode -version 1.0
+
+# Ignore self-signed certificates
+add-type @"
+    using System.Net;
+    using System.Security.Cryptography.X509Certificates;
+    public class TrustAllCertsPolicy : ICertificatePolicy {
+        public bool CheckValidationResult(
+            ServicePoint srvPoint, X509Certificate certificate,
+            WebRequest request, int certificateProblem) {
+            return true;
+        }
+    }
+"@
+[System.Net.ServicePointManager]::CertificatePolicy = New-Object TrustAllCertsPolicy
+
+$url = "https://localhost:5001"

```

Comment:
Lets rename it as urlDomain? It is fine to leave it as hard coded for now, but we probably will have to let user to input it via CLI later on (when we will have some part of project in another domain also after deploying to prod). You can change it now, if you want to.

Reply:
Okay I see how you do it right after this line, lets leave it for now hardcoded. Still it would be nice to rename the variable.

Reply:
It's hard coded because the localhost endpoint same all the time. And if user chooses local env it's always going to be this URL. But i'll rename it. And if user chooses external endpoint we are going to reassign the value.

Reply:
Renamed.